template/src/screens/IntroScreen/IntroScreen.tsx

```diff
@@ -1,41 +1,28 @@
 import React, { FC } from 'react';
-import { ImageBackground, StatusBar, StyleSheet } from 'react-native';
-import { NavigationScreenProps } from 'react-navigation';
-import styled from '@emotion/native';
 
 import { Button } from '../../components/Button';
+import { Container } from '../../components/Container';
 import { Screen } from '../../components/Screen';
 import { Text } from '../../components/Text';
-import { Container } from '../../components/Container';
-import { colors } from '../../styles';
-import bgImage from '../../assets/images/background.png';
-
-const BackgroundImage = styled(ImageBackground)`
-  ${StyleSheet.absoluteFillObject};
-`;
 
 /**
  * First screen a logged out user sees, welcoming them to the app.
  */
-export const IntroScreen: FC<NavigationScreenProps> = ({ navigation }) => {
+export const IntroScreen: FC<{ navigation: any }> = ({ navigation }) => {
```

Comment:
`import { NavigationScreenProps } from 'react-navigation';` was lighting up because `react-navigation` isn't a package anymore. I haven't figured out yet what this type should be, but I also plan to come back and refactor this to use a hook instead of the screen props